WordPress/Classes/ViewRelated/NUX/SigninKeyboardResponder.swift

```diff
@@ -148,7 +148,17 @@ extension SigninKeyboardResponder where Self: NUXAbstractViewController {
         if keyboardFrame.maxY > UIScreen.main.bounds.size.height {
             return view.frame.height - keyboardFrame.minY
         }
-        return keyboardFrame.height
+        var bottomAdjust: CGFloat = 0
+        var heightDelta = keyboardFrame.height
+        if #available(iOS 11, *) {
+            bottomAdjust = view.safeAreaInsets.bottom
+        }
+
+        if keyboardFrame.height > bottomAdjust {
```

Comment:
Is there ever a situation where `keyboardFrame.height` is not greater than `bottomAdjust`?  I couldn't seem to find that condition while testing.  Maybe there isn't and we can do away with the conditional?

Reply:
Actually, if we can do away with the conditional maybe this could be simplified to 
```
var bottomAdjust: CGFloat = 0
if #available(iOS 11, *) {
    bottomAdjust = view.safeAreaInsets.bottom
}
return keyboardFrame.height - bottomAdjust
```
and save a couple of lines?  It seemed like the value of `bottomAdjust` is either `34` on the X or `0` on everything else.   Just a thought.

Reply:
Yeah, that definitely works. Missed that last potential refactor 😉

Thanks Eric—wherever you are 😜